k8s-bench/tasks/resize-pvc/task.yaml

```diff
@@ -0,0 +1,6 @@
+script:
+- prompt: "There is a pod with name \"storage-pod\". Without creating new resources, can you resize the persistent volume claim (PVC) to request 15Gi of storage for this pod?"
```

Comment:
We should try to describe intent as close as possible to how an end-user will express.

Suggestions:

```
resize the storage to 15Gi for the storage-pod
```

or 

```
resize the storage volume to 15Gi for the storage-pod
```

```
resize the storage volume to 15Gi for the storage-pod in `resize-pv` namespace
```

```
resize the persistent volume to 15Gi for the storage-pod in `resize-pv` namespace
```